config/locales/es.yml

```diff
@@ -0,0 +1,11 @@
+es:
+  ibandit:
+    invalid_country_code: "'%{country_code}' no es un código de país ISO 3166-1 válido"
+    invalid_check_digits: "Los dígitos de control no han pasado la verificación de módulo. Se esperaba: '%{expected_check_digits}', recibido '%{check_digits}'."
+    invalid_length: "La longitud no casa con la specificación SWIFT (Se esperaba %{expected_length} símbulos, se ha recibido %{length})"
```

Comment:
Hmm, google translate seems keen on 'casa'. @Sinjo - what's your view? Also, should `specificación` be `especificación`?